packages/api/src/apps/api/app.ts

```diff
@@ -21,31 +21,44 @@ async function run(env: ProcessEnv) {
   // state shared between services
   const libs: Libs = {
     provider,
+    coingecko: new Coingecko(),
     blocks: tables.blocks.JsMap(),
     emps: {
       active: tables.emps.JsMap("Active Emp"),
       expired: tables.emps.JsMap("Expired Emp"),
     },
+    prices: {
+      usd: {
+        latest: {},
+      },
+    },
     lastBlock: 0,
     lastBlockUpdate: 0,
     registeredEmps: new Set<string>(),
+    collateralAddresses: new Set<string>(),
+    syntheticAddresses: new Set<string>(),
   };
   // services for ingesting data
   const services = {
     blocks: Services.Blocks({}, libs),
     emps: Services.Emps({}, libs),
     registry: Services.Registry({}, libs),
+    prices: Services.Prices({}, libs),
   };
 
   // services consuming data
   const actions = Actions({}, libs);
 
-  // expose calls through express
-  await Express({ port: Number(env.EXPRESS_PORT) }, actions);
-
   // warm caches
   await services.registry();
+  console.log("Got all emp addresses");
```

Comment:
Did you mean to leave these logs?

Reply:
yea it could take a minute or so to start up, so this lets you know something is happening.